news/coq-8-10beta3-is-out

```diff
@@ -0,0 +1,22 @@
+<#def ID>coq-8-10beta3-is-out</#def>
+<#def TITLE>Coq 8.10+beta3 is out</#def>
+<#def AUTHOR>Vincent Laporte</#def>
+<#def DATE>September 16th, 2019</#def>
+<#def DESCR>
+<p>The <a href="https://github.com/coq/coq/releases/tag/V8.10+beta3">third
+beta release of Coq 8.10</a> is available for testing.</p>
+
+<p>This third β version includes various bug fixes, including:</p>
+<ul>
+<li>improved warning on coercion path ambiguity;</li>
```

Comment:
Are the first two items really bug fixes?

Reply:
What do you suggest? Do not mention them? Use a more general word (e.g., “changes” instead of “bug fixes”)?

Reply:
I'd use a more general word. Maybe "bug fixes and improvements".

Reply:
done.